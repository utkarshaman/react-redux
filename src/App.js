import React from 'react';

import './App.css';


function App() {
  return (
    <div>
      Here be dragons
    </div>
  );
}

export default App;
