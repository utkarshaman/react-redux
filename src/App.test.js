import React from 'react';
import { render } from '@testing-library/react';
import App from './App';


test('renders todo', () => {
  const { getByText } = render(<App />);
  const todoElement = getByText(/here be dragons/i);

  expect(todoElement).toBeInTheDocument();
});
