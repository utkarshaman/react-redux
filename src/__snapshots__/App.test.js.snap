// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> render without crashing 1`] = `
<div>
  <getRates />
</div>
`;

exports[`<CurrencyTable /> render Currency Table 1`] = `
<div
  className="currencyTable"
>
  <table>
    <thead>
      <tr>
        <th>
          Currency
        </th>
        <th>
          Rate
        </th>
      </tr>
    </thead>
    <tbody />
  </table>
</div>
`;

exports[`<getRates /> render getRates 1`] = `<getRates />`;
